index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+  
+
+
+  <div>
+    <header class="nav">
+    <a href="" class="m-logo" >My Logo</a>
+    <nav>
+    <ul>
+         
+         <li class="link-right">
+              
+              <a href="/someurl" class="inline"><b><img src="images/menu.PNG" alt="ss"></b></a>
+          </li>
+        
+         </ul>
+         </nav>
+     </header>
+
+
+
+        <section class="section1">
+          
+          <img src="images/l1.PNG" alt="abc" class="l1">
+          <img src="images/m.PNG" alt="abc" class="m">
+          <img src="images/o.PNG" alt="abc" class="m">
+          <img src="images/r.png"  alt="abc" class="image3 dkl" >
+
+        
+
+  
+        </section>
+
+
+
+
+
+
+
+
+        <div class="section2">
+          
+          <h1 class="h2 imelex">I’m Aziz un Nabi </h1>
+                        <h1 class="h2 imelex gl">Glad to see you!</h1>
+                        <p  class="sec3p1">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                        <h5 class="ets" >ETS CONNECT</h5>
+                          
```

Comment:
- [ ] Great job 👏🏻 , I can see that the main title is an `h1` tag and subtitle  let's improve this now 👍🏻 

  * It should look like this:
```
<section class="section">
   <div class="heading-div">
       <h1>I'M ABUBKAR</h1>
       <h2>GLAD TO SEE YOU!</h2>
       <p  class="sec3p1">I’m a software developer! I can help you build a product , feature or website ...</p>
   </div>
   <ul>
      <li> 
         < a> <i href="#" class="fa fa-twitter"></i> </a>
      <li> 
      <li> 
         < a> <i href="#" class="fa linkedin"></i> </a>
      <li> 
        ..........
   <ul>
</section>
```

- [ ] Be careful with blank lines and indentation. Do not add blank lines, spaces, or indentations without a reason.
  - For readability, add blank lines to separate large or logical code blocks.
  - For readability, add two spaces of indentation. Do not use the tab key. 
 -  Check this list of the [HTML & CSS requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
 - This way will be semantically correct and the screen readers and other technologies will thank you.